packages/world/src/Constants.sol

```diff
@@ -37,9 +37,18 @@ uint128 constant MOVE_ENERGY_COST = 25543750000000;
 uint128 constant PLAYER_FALL_ENERGY_COST = MAX_PLAYER_ENERGY / 25; // This makes it so, with full energy, you die from a 25 block fall
 
 // Resource caps
-uint256 constant MAX_WHEAT_SEED = 5_000_000;
-uint256 constant MAX_OAK_SEED = 1_000_000;
-uint256 constant MAX_SPRUCE_SEED = 1_000_000;
+uint256 constant MAX_WHEAT_SEED = 444_971;
+uint256 constant MAX_MELON_SEED = 9_122;
+uint256 constant MAX_PUMPKIN_SEED = 1_717;
+
+uint256 constant MAX_OAK_SEED = 44_745;
+uint256 constant MAX_BIRCH_SEED = 35_656;
+uint256 constant MAX_JUNGLE_SEED = 27_671;
+uint256 constant MAX_SAKURA_SEED = 3_437;
+uint256 constant MAX_ACACIA_SEED = 20_701;
+uint256 constant MAX_SPRUCE_SEED = 19_594;
+uint256 constant MAX_DARK_OAK_SEED = 13_732;
+uint256 constant MAX_MANGROVE_SEED = 21_610;
```

Comment:
nit: if we rename the object types to sapling, should we call these `_SAPLING` too?

Reply:
(or keep the seed name for the object types, no strong opinion)

Reply:
yep good catch, I think it might be better to use sapling, as they work different from seeds